### PR TITLE
Implements test for the resource view.

### PR DESCRIPTION
**Pull request by boyan-tonchev**
## What
Tests for the resource view have been implemented.

## Why
We want to replace the old implementation of YASGUI with the new one, but there were no tests for this view. Implementing tests will make it easier to verify that the changes do not break the implementation.

## How
Implemented tests.

### CODE REVIEW

File: test-cypress/integration/resource/resource.spec.js [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+
+const FILE_TO_IMPORT = 'resource-test-data.ttl';
+const SUBJECT_RESOURCE = 'http:%2F%2Fexample.com%2F%2320';
+const SUBJECT_RESOURCE_SHORT_URI = 'ex:20';
+const PREDICATE_SOURCE = 'http:%2F%2Fwww.w3.org%2F2000%2F01%2Frdf-schema%23label';
+const CONTEXT_EXPLICIT = 'http://www.ontotext.com/explicit';
+const OBJECT_RESOURCE = 'http:%2F%2Fexample.com%2F%23City';
+const IMPLICIT_EXPLICIT_RESOURCE = 'http:%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23type';
+
+describe('', () => {
```
Comment (svilenvelikov) [inline test-cypress/integration/resource/resource.spec.js:14]: put a description

File: test-cypress/steps/explore/graphs-visualizations-steps.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const VIEW_URL = '/graphs-visualizations';
+
+export class GraphsVisualizationsSteps {
```
Comment (svilenvelikov) [inline test-cypress/steps/explore/graphs-visualizations-steps.js:3]: There is a VisualGraphSteps already

File: test-cypress/integration/resource/resource.spec.js (hunk 1 above)
Comment (boyan-tonchev) [reply test-cypress/integration/resource/resource.spec.js:14]: Ups

File: test-cypress/steps/explore/graphs-visualizations-steps.js (hunk 2 above)
Comment (boyan-tonchev) [reply test-cypress/steps/explore/graphs-visualizations-steps.js]: Fixed.

File: test-cypress/integration/resource/resource.spec.js (hunk 1 above)
Comment (boyan-tonchev) [reply test-cypress/integration/resource/resource.spec.js]: Done

**Review by svilenvelikov — APPROVED** [review]

**Comment by boyan-tonchev** [comment]
Jenkins retry build

**Comment by boyan-tonchev** [comment]
Jenkins retry build

**Comment by boyan-tonchev** [comment]
Jenkins retry build

**Comment by boyan-tonchev** [comment]
Jenkins retry build

**Pull request merged**